components/server/src/server.rs

```diff
@@ -268,9 +269,12 @@ struct TikvServer<ER: RaftEngine, F: KvFormat> {
     check_leader_worker: Worker,
     sst_worker: Option<Box<LazyWorker<String>>>,
     quota_limiter: Arc<QuotaLimiter>,
-    causal_ts_provider: Option<Arc<CausalTsProviderImpl>>, // used for rawkv apiv2
+    causal_ts_provider: Option<Arc<CausalTsProviderImpl>>,
+    // used for rawkv apiv2
```

Comment:
Emmm, should this commend be for the previous line?

Reply:
This format tool is so weird